### PR TITLE
chore: sort dependencies in setup.py

### PR DESCRIPTION
**Pull request by jsam**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -247,14 +247,14 @@ def _get_disribution_url():
     author_email="contact@datascience.ch",
     url="https://github.com/swissdatasciencecenter/renku-python",
     project_urls={
-        "Changelog": ("https://github.com/swissdatasciencecenter/renku-python" "/blob/master/CHANGES.rst"),
+        "Changelog": "https://github.com/swissdatasciencecenter/renku-python" "/blob/master/CHANGES.rst",
```
Comment (Panaetius) [inline setup.py:250]:
```suggestion
        "Changelog": "https://github.com/swissdatasciencecenter/renku-python/blob/master/CHANGES.rst",
```

File: setup.py [hunk 2]
```diff
@@ -150,13 +149,14 @@ def run(self):
     extras_require["nodocs"].extend(reqs)
 
 install_requires = [
-    "appdirs>=1.4.3,<=1.4.4 ",
+    "PyYAML>=3.12,<=5.3.1",
```
Comment (Panaetius) [inline setup.py:152]: Did you use isort for this? I wonder if case-insensitive sort would be better. At least I could see myself searching for pyyaml and being confused when it's not with the other `py*` entries. But just a minor detail

**Review by Panaetius — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (jsam) [reply setup.py]: Fair point. Used case-insensitive sort.

**Review by Panaetius — APPROVED** [review]
:rocket:

**Review by Panaetius — APPROVED** [review]

**Pull request merged**